### ISSUE TITLE
[Feature request] Wear

### ISSUE THREAD

**Issue by scarral** [MilosKozak/AndroidAPS#2934]
I absolutely love AAPS v2 watchface, but I keep missing some information that I think would be very useful. 

- Insulin activity curve,

- day of the week,

- When I installed NSClient instead of AAPS in my phone (my son is the PWD, I just follow) BG predictions are not shown anymore, although they were shown in AAPS. It would be nice to have them in NSClient as well. 

Thanks! 

Reporting bugs
--------------
- **Note the precise time the problem occurred** and describe the circumstances and steps that caused
  the problem
- Note the Build version (found in the About dialog in the app, when pressing the three dots in the
  upper-right corner).
- Obtain the app's log files, which can be found on the phone in
  _/storage/emulated/0/Android/data/info.nightscout.androidaps/_
  See https://github.com/MilosKozak/AndroidAPS/wiki/Accessing-logfiles
- Open an issue at https://github.com/MilosKozak/AndroidAPS/issues/new

**Comment by reijoahola-nightscout** [comment]
Strange, I have installed NSClient (2.7.0-rc4) in my second (test) phone and it shows predictions. I rememember that it has worked also earlier, not only in 2.7.0-rc.

**Comment by scarral** [comment]
I've tried disabling and enabling "show predictions in watchface" and restarting the phone several times and I still cannot see the predictions in my watchface. They worked just fine when I had AAPS installed in my phone instead of NSClient. My phone is a Samsung Galaxy A3 with AndroidOS v. 8.0 and my watch is a Fossil Sport.

**Comment by reijoahola-nightscout** [comment]
Sandra > I've tried disabling and enabling "show predictions in watchface" and restarting the phone several times and I still >cannot see the predictions in my watchface (they are shown in the Overview of NSClient in my phone, but not in my >watchface). They worked just fine when I had AAPS installed in my phone instead of NSClient. My phone is a Samsung Galaxy >A3 with AndroidOS v. 8.0 and my watch is a Fossil Sport.

Now I noticed my misunderstanding. I checked how predictions are shown in NSClient's own (main) display. You seem to think about watchface when NSClient is used. I do not have a test system where watch is connected into NSClient.

**Comment by scarral** [comment]
Yes in the watchface, that's why the title of this feature request is "Wear", and I started with "I absolutely love AAPS v2 watchface, but..." :)

**Comment by rICTx-T1D** [comment]
If you like to see the weekday, you can use my new watchface based on AAPSv2. Check out from https://github.com/rICTx-T1D/NightScoutAndroidAPS-fork/tree/rICTx-T1D_watchface01/ or watch on Pull request https://github.com/nightscout/AndroidAPS/pull/14
